.github/pull_request_template.md

```diff
@@ -10,4 +10,5 @@ Resolves #XXX
 
 - [ ] Relevant documentation (e.g. READMEs, Technical Foundation) updated
 - [ ] Issue AC are copied to this PR & are met
+- [ ] Manual Browser Testing performed (via local browser or BrowserStack w/ modheader)
```

Comment:
Moving modifier to beginning (I think this is what you mean)

```suggestion
- [ ] Manual Browser Testing performed (w/ modheader, locally or in BrowserStack)
```

Reply:
```suggestion
- [ ] Manual Browser Testing performed (with modheader, either locally or on BrowserStack)
```